src/admin-dhis2/components/AdminDhis2DataElementForm/AdminDhis2DataElementForm.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useMemo, useEffect, useState } from 'react';
+import { toast } from 'react-toastify';
+
+import getService from '../../../react-components/utils/angular-utils';
+import { SearchSelect } from '../../../requisition-order-create/search-select'
+
+function AdminDhis2DataElementForm({onSubmit, onCancel, refetch, serverId, datasetId}) {
```

Comment:
`function AdminDhis2DataElementForm({ onSubmit, onCancel, refetch, serverId, datasetId }) {`

Reply:
simply add the white space after and before curly bracket.

Reply:
updated